src/posts/2023-05-30-tietoevry/index.md

```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-05-30T15:56:00.000Z'
+title: 'Better policies, less tool sprawl: Tietoevry uses Cilium and Hubble'
+ogImage: ogimage.webp
+ogSummary: 'Learn about how Tietoevry uses Isovalent Enterprise for Cilium with Hubble to have advanced network policies'
```

Comment:
This makes it sound like Hubble allows them to have network policy, can you rewrite to make it more clear?